pyroSAR/gamma/srtm.py

```diff
@@ -142,6 +143,101 @@ def transform(infile, outfile, posting=90):
     par2hdr(outfile + '.par', outfile + '.hdr')
 
 
+def dem_autocreate(geometry, demType, outfile, buffer=0.01, logpath=None, username=None, password=None):
+    """
+    | automatically create a DEM in Gamma format for a defined spatial geometry
+    | the following steps will be performed:
+
+    - collect all tiles overlapping with the geometry
+
+      * if they don't yet exist locally they will automatically be downloaded
+      * the tiles will be downloaded into the SNAP auxdata directory structure, e.g. $HOME/.snap/auxdata/dem/SRTM 3Sec
+
+    - create a mosaic GeoTiff of the same spatial extent as the input geometry plus a defined buffer using gdalwarp
+    - subtract the EGM96-WGS84 Geoid-Ellipsoid difference and convert the result to Gamma format using Gamma command srtm2dem
+    
+      * this correction is not done for TanDEM-X data, which contains ellipsoid heights; see `here <https://geoservice.dlr.de/web/dataguide/tdm90>`_
+
+    Parameters
+    ----------
+    geometry: spatialist.vector.Vector
+        a vector geometry delimiting the output DEM size; CRS must be WGS84 LatLon (EPSG 4326)
+    demType: str
+        the type of DEM to be used; see :func:`~pyroSAR.auxdata.dem_autoload` for options
+    outfile: str
+        the name of the final DEM file
+    buffer: float
+        a buffer in degrees to create around the DEM file
```

Comment:
Should be: "around geometry" and not "around DEM file" ?

Reply:
sure thing. fixed by 4fd01e6